addons/website/static/tests/tours/snippet_social_media.js

```diff
@@ -89,9 +89,9 @@ registerWebsitePreviewTour('snippet_social_media', {
     url: '/',
     edition: true,
 }, () => [
-    ...dragNDrop({id: 's_social_media', name: 'Social Media'}),
-    ...clickOnSnippet({id: 's_social_media', name: 'Social Media'}),
-    ...addNewSocialNetwork(7, 7, 'https://www.youtu.be/y7TlnAv6cto'),
+    ...wTourUtils.dragNDrop({id: 's_social_media', name: 'Social Media'}),
+    ...wTourUtils.clickOnSnippet({id: 's_social_media', name: 'Social Media'}),
+    ...addNewSocialNetwork(8, 8, 'https://www.youtu.be/y7TlnAv6cto'),
```

Comment:
Maybe we should dragAndDrop discord icon lilke we add other icons.

Reply:
Hello @msh-odoo i am not sure what is expected here, but on dragAndDroping the `s_social_media` will definitely have the discord icon there hence the count adds up to 8. Or is there any other context are you talking about ?